logger.go

```diff
@@ -0,0 +1,34 @@
+package cron
+
+import "log"
+
+// Logger is the simplest interface for logging
+type Logger interface {
+	Logf(format string, args ...interface{})
```

Comment:
I think it would be more approperiate if you use `Printf(format string, args ...interface{})` instead.